cmake/ValhallaConfigPyProject.cmake

```diff
@@ -0,0 +1,33 @@
+# Usage:
```

Comment:
we should probably make it obvious to new contributors to use the `-P` flag when configuring the project, if they want to work on any Python code, maybe over in CONTRIBUTING.md

Reply:
it's in the python readme. I thought it fits better there